src/Components/Votes/Votes.jsx

```diff
@@ -0,0 +1,50 @@
+import { incrementVotes, decrementVotes } from "../../Utils/api";
+import { useState } from "react";
+import "./Votes.css";
+
+function Votes({ article, setArticle, article_id }) {
+  const [voted, setVoted] = useState(false);
+
+  function handleClick() {
+    if (voted === false) {
+      setVoted(true);
+      setArticle((article) => {
+        return { ...article, votes: article.votes + 1 };
+      });
+      incrementVotes(article_id)
+        .then(() => {
+          return { ...article, votes: article.votes - 1 };
+        })
+        .catch((err) => {
+          if (err) {
+            return <p>Unable to vote right now, try again later.</p>;
+          }
+        });
+    } else {
+      setVoted(false);
+      setArticle((article) => {
+        return { ...article, votes: article.votes - 1 };
+      });
+      decrementVotes(article_id)
+        .then(() => {
+          return { ...article, votes: article.votes + 1 };
+        })
+        .catch((err) => {
+          if (err) {
+            return <p>Unable to vote right now, try again later.</p>;
+          }
+        });
+    }
+  }
+
+  return (
+    <div className="vote-buttons-container">
+      <button className="vote-button" onClick={handleClick}>
```

Comment:
seeing as if a user clicks on this again to remove their vote we are going to remove the vote they've just added, it might be good to style this in a way that when they first click it, they know they have added a like and when they click it again, the styling is removed so that they can unlike - it is a visual cue to the user that once they've liked something, clicking it again will remove the like!

Reply:
Yeah, I was trying to figure that out. Managed to find a solution.